modules/distribution/product/src/main/resources/conf/infer.json

```diff
@@ -137,5 +147,13 @@
       "synapse_handlers.analytics_handler.name": "analytics_handler",
       "synapse_handlers.analytics_handler.class": "$ref{apim.analytics.handler_impl}"
     }
+  },
+  "server.disable_traffic_manager": {
```

Comment:
This should be "enable". Shall we use "management"? 

so it will be `enable_traffic_management`. wdyt?